docs/docs/schema/config.json

```diff
@@ -200,6 +212,36 @@
             "title": "PipelineConfig",
             "type": "object"
         },
+        "SchemaRegistryConfig": {
+            "additionalProperties": false,
+            "description": "Configures schema registry.",
+            "properties": {
+                "enabled": {
+                    "default": false,
+                    "description": "If the Schema Registry handler should be initialized.",
+                    "env_names": [
+                        "enabled"
```

Comment:
@disrupted Isn't it a bit strange that every field has, by default, an env variable? This might collide with other env variables in your execution environment.

Reply:
yeah, absolutely! Never knew that's the case. Shouldn't it be only the fields that specify a `KPOPS_...` env variable?

Reply:
Yeah, I thought so, too. But Pydantic creates the `env` whenever you define a Field. Maybe setting it to `None` will fix the problem, but that is not a great solution.
```python
class SchemaRegistryConfig(BaseSettings):
    """Configuration for Schema Registry."""

    enabled: bool = Field(
        default=False,
        description="If the Schema Registry handler should be initialized.",
    )
```

Reply:
very weird, what are the docs saying or is there an issue about it?

Reply:
I couldn't find anything about it 😞

Reply:
I think we should also set the `env_prefix` option https://docs.pydantic.dev/1.10/usage/settings/#environment-variable-names

Reply:
I added the `env_prefix`. I didn't get rid of the `env` in the field of the configs. This is because the doc generation is dependent on the `env`. If I remove it, the environment variables' docs will no longer be generated. For now, it is fine. We should do a follow-up PR/Issue for removing the `env`. We can also keep them.

Reply:
Beware that at least in v2 `env_prefix` is not taken into consideration for aliases

Reply:
Also, `env` doesn't exist in v2